runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java

```diff
@@ -129,12 +131,16 @@ private void waitForWorkflow(@NonNull UUID workflow, @NonNull ManagementClient c
     public void invoke() {
         for (int x = 0; x < retry; x++) {
             try {
-                runtime.invalidateLayout();
-                Layout requestLayout = new Layout(runtime.getLayoutView().getLayout());
-                ManagementClient orchestrator = getOrchestrator(requestLayout);
-                UUID workflowId = sendRequest(orchestrator);
-                waitForWorkflow(workflowId, orchestrator, timeout, pollPeriod);
-            } catch (NetworkException | TimeoutException e) {
+                Optional<ManagementClient> orchestrator = getOrchestrator();
+                if(orchestrator.isPresent()){
```

Comment:
nit: add spaces after `if` and before `{`

Reply:
Sounds good